cmd/kube-proxy/app/server_others.go

```diff
@@ -92,9 +92,14 @@ func newProxyServer(
 
 	dbus = utildbus.New()
 	iptInterface = utiliptables.New(execer, dbus, protocol)
-	ipvsInterface = utilipvs.New(execer)
 	kernelHandler = ipvs.NewLinuxKernelHandler()
 	ipsetInterface = utilipset.New(execer)
+	proxyMode := getProxyMode(string(config.Mode), iptInterface, kernelHandler, ipsetInterface, iptables.LinuxKernelCompatTester{})
+	if proxyMode == proxyModeIPVS {
+		ipvsInterface = utilipvs.New(execer)
+	} else {
```

Comment:
This else is unnecessary, it's already nil.

Reply:
@mikedanese Exactly.